src/web-ui/src/partials/AppModal/DemoWalkthrough/pages/SimilarItemRecommendations.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <DemoWalkthroughPageLayout :drawArrows="drawArrows">
+    <h1 class="heading mb-3">
+      Use case 3
+      <div class="heading-similar-item-recommendations">Similar item recommendations</div>
+    </h1>
+
+    <div :class="{ content: true, 'content--mobile': isMobile }">
+      <p class="similar-item-recommendations pl-2" ref="similar-item-recommendations">
+        The Similar item recommendations use case is implemented in all product detail pages under the “Compare similar
+        items” section.
+      </p>
+
+      <p class="demo-guide-text mb-0">
+        Go to the <span ref="demo-guide">Demo Guide</span> to learn more about the Similar Item recommendations and for
+        tips on how to demonstrate this use case during customer meetings.
```

Comment:
Lets remove the *and for tips on how to demonstrate this use case during customer meetings. * for now since this is not something that is providing info to the end customer

Reply:
Done